src/app/footer/footer.component.ts

```diff
@@ -49,10 +59,17 @@ export class FooterComponent extends Base implements OnInit {
    */
   private readonly WEBSERVICE_DOWN_STATUS_CODES = [0, 404, 502, 504];
 
-  constructor(private metadataService: MetadataService, private footerService: FooterService) {
+  constructor(private metadataService: MetadataService, private footerService: FooterService, private _snackBar: MatSnackBar) {
```

Comment:
It should just be `snackbar` -- we don't use the convention of fields starting with underscores.